ibm_catalog.json

```diff
@@ -317,25 +333,33 @@
         {
           "label": "Resource groups only",
           "name": "resource-group-only",
+          "index": 2,
           "install_type": "fullstack",
           "working_directory": "solutions/fully-configurable",
           "iam_permissions": [
             {
               "role_crns": [
-                "crn:v1:bluemix:public:iam::::role:Editor"
+                "crn:v1:bluemix:public:iam::::role:Administrator"
               ],
-              "service_name": "resource-group"
+              "service_name": "All Account Management services",
+              "notes": "Administrator access is required for all Account Management services."
             }
           ],
           "architecture": {
+             "features": [
+              {
+                "title": " ",
+                "description": "Configured to use IBM secure by default standards, but can be edited to fit your use case."
+              }
+            ],
             "diagrams": [
               {
                 "diagram": {
                   "caption": "Resource Groups",
-                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-account-infrastructure-base/main/reference-architectures/rg-only.svg",
+                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-account-infrastructure-base/refs/heads/14185-improve-DA-user-experience/reference-architectures/rg-only.svg",
```

Comment:
This reference should stay as `main` and the branch for the SVG should be merged to main to update the image that appears here.